Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -75,6 +76,10 @@ WK_SWIFT_WERROR_FLAGS_ = -Werror StrictMemorySafety;
 SWIFT_INCLUDE_PATHS = $(SRCROOT)/Modules/Internal $(SWIFT_INCLUDE_PATHS_$(USE_INTERNAL_SDK));
 SWIFT_INCLUDE_PATHS_ = $(SRCROOT)/Platform/spi/visionos $(SRCROOT)/Platform/spi/ios $(SRCROOT)/Platform/spi/Cocoa $(SRCROOT)/Platform/spi/Cocoa/Modules $(inherited);
 
+// Swift/C++ interop, by default, exports any symbols that are exposed from Swift to C++.
+// We don't want that.
+SWIFT_CXX_UNEXPORTED_SYMBOL_LDFLAGS = -Wl,-unexported_symbol,__ZN6WebKit16getSwiftLogoDataEv;
```

Comment:
Do we have a bug tracking this? Seems like it should not be a requirement that the language exports any symbol that crosses the interop boundary.

I know about rdar://116086634, not sure if it's the same thing though.